src/content/migrate/5.md

```diff
@@ -33,6 +33,8 @@ Some Plugins and Loaders might have a beta version that has to be used in order
 
 W> Check related Plugins and Loaders migration guide when upgrading across major versions.
 
+W>  ExtendedAPIPlugin is removed and the logic is merged into APIPlugin.
```

Comment:
```suggestion
W>  ExtendedAPIPlugin is removed and the logic is merged into [`APIPlugin`](https://github.com/webpack/webpack/blob/master/lib/APIPlugin.js).
```

@jeffin143 I think @montogeek meant this .

Reply:
Hey thanks :)